AuthenticationAndAuthorization/BusinessLogic/Services/JwtService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using DataAccounts;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Reflection;
+using System.Security.Claims;
+using System.Text;
+
+namespace BusinessLogic
+{
+    public class JwtService : IJwtService
```

Comment:
Implementation should be in infrastructure assembly because it's not related to business logic